src/components/blocks/FormBlock.tsx

```diff
@@ -116,7 +122,7 @@ const FormBlock: React.FC = () => {
                 label='Movie average raiting'
                 name='vote_average'
                 rules={rules.vote_average}
-                initialValue={formik.values.vote_average}
+                initialValue={formik.values?.vote_average}
```

Comment:
I'm not sure that you need `?` -as i know `formik` always provide `values` object

Reply:
made edits in task 9